liquidation-monitor/src/agent.js

```diff
@@ -0,0 +1,522 @@
+const {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  ethers,
+  getEthersProvider,
+} = require('forta-agent');
+const BigNumber = require('bignumber.js');
+
+// To-do: Replace node-fetch with axios and remove TS
+const config = require('../agent-config.json');
+const { getAbi, ts, callAPI } = require('./utils');
+
+// Stores information about each account
+const initializeData = {};
+
+ts('Starting bot');
```

Comment:
Remove these `ts` calls.

Reply:
I left it verbose to to be able to see what's going on at each step. To mute the `ts` calls, I can comment out the `console.log` line in the function to be able to turn troubleshooting and debug on an off. I think leaving some of the logging would be helpful for future iterations. What do you think?